src/Flare.purs

```diff
@@ -98,10 +92,10 @@ instance applyFlare :: Apply Flare where
 instance applicativeFlare :: Applicative Flare where
   pure x = Flare [] (pure x)
 
--- | The main data type for a Flare UI. It encapsulates the `Eff` action
+-- | The main data type for a Flare UI. It encapsulates the `Effect` action
 -- | which is to be run when setting up the input elements and corresponding
 -- | signals.
-newtype UI e a = UI (Eff (dom :: DOM, channel :: CHANNEL | e) (Flare a))
+newtype UI e a = UI (Effect (Flare a))
```

Comment:
We should remove the first type argument (`e`) here. It was used to inject other effects but it should not be necessary anymore with the new `Effect` mechanism. This will probably also allow us to remove a few `forall e.`'s elsewhere.

Reply:
I've removed the leftover `e`s.  Good catch!